pkg/scheduler/apis/config/types.go

```diff
@@ -190,6 +190,9 @@ type Plugins struct {
 	// Filter is a list of plugins that should be invoked when filtering out nodes that cannot run the Pod.
 	Filter *PluginSet
 
+	// PostFilter is a list of plugins that are invoked after filtering phase.
```

Comment:
pls indicate that they will be invoked on both success and failure of the filtering phase.

Reply:
done.